.changeset/short-jokes-promise.md

```diff
@@ -0,0 +1,11 @@
+---
+'@livekit/rtc-node-linux-arm64-gnu': minor
+'@livekit/rtc-node-win32-x64-msvc': minor
+'@livekit/rtc-node-linux-x64-gnu': minor
+'@livekit/rtc-node-darwin-arm64': minor
+'@livekit/rtc-node-darwin-x64': minor
+'livekit-server-sdk': minor
+'@livekit/rtc-node': minor
```

Comment:
I don't think this change affects the `@livekit/rtc-node` packages (yet)? 
we should remove all lines apart from `livekit-server-sdk`

Reply:
Yes, I was not sure why it marked it as changed. It's probably because of lock file. I'll remove them from the changeset then.